### ISSUE TITLE
`SymbolMaster.overrideProperties` is an empty array

### ISSUE THREAD

**Issue by mathieudutour** [sketch-hq/sketch-file-format-ts#8]

**Comment by jedrichards** [comment]
Making a note that when populated it should have content like this

```
"overrideProperties": [
  {
    "_class": "MSImmutableOverrideProperty",
    "canOverride": false,
    "overrideName": "1A9A380A-E929-4AEC-91B2-4377EDB4E3AE_stringValue"
  }
]
```

**Comment by jedrichards** [comment]
Fixed in 4.0.0
